### DISCUSSION TITLE
Slideshow and Video Recording

### DISCUSSION THREAD

**Discussion by ming-cui** [quarto-dev/quarto-cli#5623] [Q&A] [answered]
Hi, I'm looking for a way to do multi-take video recording on individual pages of a reveal.js slide presentation. Specifically, I would like to be able to record my narration for a given slide page multiple times until I'm satisfied with the recording. Ideally, this could be enabled through a plugin for reveal.js or another tool that integrates with it, but please let me know if there are any third-party tools can do that. Thanks!

**Comment by mcanouil** [comment, marked as answer]
I don't believe it is in Quarto's scope.

If there is a third party plugin or else, then you can integrate it yourself into a Quarto extension, see <https://quarto.org/docs/extensions/creating.html>.

**Reply by ming-cui** [reply]
Thank you for your response! I'll be looking for relevant plugins in reveal.js community.
